language_model.py

```diff
@@ -0,0 +1,69 @@
+from functools import partial
+from multiprocessing import Pool
+from typing import List
+
+import numpy as np
+import scipy.sparse as sp
+from tqdm import tqdm
+
+import utility
+import query_handling
+
+count_vectorizer = sp.load_npz("Generated Files/count_vec_matrix.npz")
+
+doc2word = utility.load_vector_file("Generated Files/doc2word.csv")
+word2vec = utility.load_vector_file("Generated Files/word2vec.csv")
+inverse_w2v = {v: k for k, v in word2vec.items()}
+dirichlet_prior = sum([len(i) for i in list(doc2word.values())]) / len(doc2word)
```

Comment:
Kommer denne Dirichlet prior ligning fra artiklen? Bliver det ikke en værdi over 1, hvilket ikke giver mening som prior vægt?

Reply:
Kan se fra @slayeren's afsnit at det måske er bedre at ændre navnet til f.eks.:
```suggestion
dirichlet_smoothing = sum([len(i) for i in list(doc2word.values())]) / len(doc2word)
```
Eller noget lignende.

Reply:
make sure to rename via python, if we rename.